lib/ClangImporter/ClangImporter.cpp

```diff
@@ -865,46 +865,37 @@ ClangImporter::create(ASTContext &ctx,
     }
   }
 
-  // FIXME: These can't be controlled from the command line.
-  llvm::IntrusiveRefCntPtr<clang::DiagnosticOptions> diagnosticOpts{
-    new clang::DiagnosticOptions
-  };
+  {
+    // Unfortunately these can't be controlled from the command line, because
+    // they're used to /parse/ the command line.
```

Comment:
Similarly here, a mention that this is called "temp" because we're going to be resetting it down below, with a pointer to where, will help future readers.